<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatViewSearchBar</class>
 <widget class="QWidget" name="ChatViewSearchBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="hideButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/16x16/actions/oxygen/16x16/actions/application-exit.png</normaloff>:/16x16/actions/oxygen/16x16/actions/application-exit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClearableLineEdit" name="searchEditLine"/>
     </item>
     <item>
      <widget class="QToolButton" name="searchDownButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/16x16/actions/oxygen/16x16/actions/go-down.png</normaloff>:/16x16/actions/oxygen/16x16/actions/go-down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="searchUpButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/16x16/actions/oxygen/16x16/actions/go-up.png</normaloff>:/16x16/actions/oxygen/16x16/actions/go-up.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="caseSensitiveBox">
     <property name="text">
      <string>case sensitive</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="searchSendersBox">
     <property name="text">
      <string>search nick</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="searchMsgsBox">
     <property name="text">
      <string>search message</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="searchOnlyRegularMsgsBox">
     <property name="text">
      <string>ignore joins, parts, etc.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">clearablelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
